tempesta_fw/ss_skb.c

```diff
@@ -863,7 +858,14 @@ ss_skb_get_room(struct sk_buff *skb_head, struct sk_buff *skb, char *pspt,
 	int r = skb_fragment(skb_head, skb, pspt, len, it);
 	if (r == len)
 		return 0;
-	return r;
+	/*
+	 * TODO: skb_fragment() returns a number of processed data, which can
```

Comment:
Well, there is no such direct scenario, because `ss_skb_get_room()` is called with `len > 0` and `skb_fragment()` now is never called by other modules. But when it's called, the `len` can be modified e.g. here (if `len < 0`): https://github.com/tempesta-tech/tempesta/blob/8d90875697968ddbeccdcdc008503f9fe2e290e4/tempesta_fw/ss_skb.c#L803
and `skb_fragment()` will return a value different from the `|len|`.

The function removed in this PR as dead one -- `tfw_http_msg_del_eol()` -- could do that. 

Now this never happens, but I decided to add this return value modifications to prevent such cases.

Reply:
So do we really need `TODO` something on this as pointed out in the comment? What about just to add `WARN_ON()` instead?

Reply:
OK, but it seems there is nothing to do for now, so please just remove `TODO` from the comment.